src/Sulu/Bundle/MediaBundle/DependencyInjection/Configuration.php

```diff
@@ -133,6 +133,7 @@ public function getConfigTreeBuilder()
             ->arrayNode('format_cache')
                 ->addDefaultsIfNotSet()
                 ->children()
+                    ->scalarNode('private_path')->defaultValue('%kernel.project_dir%/var/images-cache')->end()
```

Comment:
Somebody maybe ask why I did choose to store it here. There would be two alternatives to this but both make more problems.

Alternative *A*  would be to store it in `var/cache/` directory. This I find not good as many developers remove that cache folder and I think its not good to trigger the rerender images again.

Alternative *B*  would be to store it in `var/uploads` directory. What problem I have with this is that it is mostly be the folder which is also be backed up, as in projects I mostly have other files also there for example:

 - `var/uploads/media`
 - `var/uploads/pdfs`
 - `var/uploads/invoices`
 - `var/uploads/wallets`

Things which need to be backup. Thats why I decided to add a additional folder `var/images` in the root folder. So I can still make whole `var/uploads` part of the backup and don't need to filter out `var/uploads/images` folder.

To more represent that its just a cache I decided to name it `var/images-cache`.

Reply:
After discussion with @chirimoya it was decided to go with `var/media/formats` and also change the urls and folder in the skeleton for the public to `public/media/formats` and `/media/formats/{slug}`